instrumentation/opentelemetry-instrumentation-annotations-1.16/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/otelannotations/WithSpanInstrumentationModule.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.otelannotations;
+
+import static java.util.Collections.singletonList;
+
+import application.io.opentelemetry.instrumentation.annotations.WithSpan;
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.List;
+
+/** Instrumentation for methods annotated with {@link WithSpan} annotation. */
+@AutoService(InstrumentationModule.class)
+public class WithSpanInstrumentationModule extends InstrumentationModule {
+
+  public WithSpanInstrumentationModule() {
+    super("opentelemetry-annotations");
```

Comment:
Just to make sure: this is the same name as the extension annotations, is that correct? (I think it makes sense for these two modules to have the same name, just wanted to confirm that this is on purpose, and not copy-pasta)

Reply:
It was definitely copy-pasta, I will rename to match the module names